test/replcompletions.jl

```diff
@@ -390,6 +390,23 @@ c, r, res = test_scomplete(s)
         @test s[r] == "tmpfoob"
         rm(dir)
     end
+
+    # Tests detecting of files in the env path (in shell mode)
+    let
+        oldpath = ENV["PATH"]
+        ENV["PATH"] = homedir()
+        file = joinpath(path, "tmp-executable")
```

Comment:
@yuyichao I was thinking I should at least revise the test I added to cleanup on failure.